lib/rom/repository.rb

```diff
@@ -1,58 +1,188 @@
+require 'addressable/uri'
+
 module ROM
-  # Repository exposes schema if supported by the adapter
+  # Abstract adapter class
```

Comment:
@solnic doc in `ROM::Repository` still refers to `adapter`. Would it be better to refer to repository or make it clear that an adapter is providing this repository?